packages/ses/src/commons.js

```diff
@@ -220,6 +242,19 @@ export const immutableObject = freeze(create(null));
  */
 export const isObject = value => Object(value) === value;
 
+/**
+ * isError tests whether an object descends from the intrinsic Error
+ * constructor.
+ * We capture the original error constructor as FERAL_ERROR to provide a clear
+ * signal for reviewers that we are handling an object with excess authority
+ * that we are carefully hiding from client code, like stack trace inspection.
```

Comment:
```suggestion
 * signal for reviewers that we are handling an object with excess authority,
 * like stack trace inspection, that we are carefully hiding from client code.
```
avoids a parsing ambiguity.